src/pykiso/cli.py

```diff
@@ -89,13 +89,19 @@ def check_file_extension(click_context: click.Context, param: click.Parameter, p
     return paths
 
 
-def active_threads() -> list[str]:
+def active_threads() -> list[list[str]]:
     """Get the names of all active threads except the main thread."""
```

Comment:
Now you also return where the code is running. so the docstrings needs to be adapted

Reply:
done